gnuradio-runtime/include/gnuradio/realtime.h

```diff
@@ -12,10 +12,16 @@
 #define INCLUDED_GR_REALTIME_H
 
 #include <gnuradio/api.h>
-#include <gnuradio/realtime_impl.h>
 
 namespace gr {
 
+typedef enum { RT_OK = 0, RT_NOT_IMPLEMENTED, RT_NO_PRIVS, RT_OTHER_ERROR } rt_status_t;
+
+enum rt_sched_policy {
```

Comment:
Maybe switch to `enum class` while you're at it, for better type safety?

Reply:
I also prefer enum classes, but that's an API break that we might be able to avoid.